hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java

```diff
@@ -1056,6 +1061,25 @@ private String reconfDiskBalancerParameters(String property, String newVal)
     }
   }
 
+  private String reconfSlowIoWarningThresholdParameters(String property,
+      String newVal) throws ReconfigurationException {
```

Comment:
your code isn't formatted properly I believe, the line wrap is at 100
```
  private String reconfSlowIoWarningThresholdParameters(String property, String newVal)
      throws ReconfigurationException {
```

can use the hadoop formatter if you need & if you use intelliJ
https://github.com/apache/hadoop/blob/trunk/dev-support/code-formatter/hadoop_idea_formatter.xml

Reply:
Thanks! I have modified the settings.